docs/INSTALLATION.md

```diff
@@ -64,6 +64,35 @@ of the Microsoft Visual C++ compiler.
 
 You should be able to use other integrated IDE's such as JetBrains' CLion in a similar manner.
 
+### Installing Boost on Windows
+
+The `meetup` and `gigasecond` exercises depend on the [Boost](https://www.boost.org/) libraries. The following steps
+describe how to set it up.
+
+The steps assume you install Boost under `C:\Program Files\Boost`, writing here requires Administrator privileges. You
```

Comment:
I think it'd be more appropriate to suggest a folder in the user's personal files. Alternatively, putting it into something such as `c:\libs` is also common.

Reply:
I removed any mention of an installation folder.